### ISSUE TITLE
Format error with save as HTML and PDF 

### ISSUE THREAD

**Issue by kingspp** [PHPOffice/PHPWord#562]
// Saving the document as HTML file...
$objWriter = \PhpOffice\PhpWord\IOFactory::createWriter($phpWord, 'HTML');
$objWriter->save('helloWorld.html');

I have even tried it with tcpdf api
\PhpOffice\PhpWord\Settings::setPdfRendererPath('tcpdf');
\PhpOffice\PhpWord\Settings::setPdfRendererName('TCPDF');

I am able to output file.html and file.pdf, but the formatting is not present. The given format is applied only when it is saved as .docx file. Otherwise the format does not work on html and pdf files!!

**Comment by prabhakaran8737** [comment]
@kingspp did you manage to fix the formatting issue because me too hang up with the same issue now. Have you found any solution?

**Comment by kingspp** [comment]
No, solution using PHPWord framework. You can use xpdf and convert word to PDF in linux using shell_exec();

**Comment by prabhakaran8737** [comment]
@kingspp Can you paste the installation documentation here, it will be helpful
